submissions/Ant-C-tech/oop-exercise/index.js

```diff
@@ -0,0 +1,126 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(name, friends) {
+        this.name = name;
+        this.friends = friends;
+        this.templete = `My name is ${this.name}`;
+    };
+
+    getVoice() {
+        let pluralEnding;
+        (this.friends && this.friends.length > 1) ? pluralEnding = 's': pluralEnding = '';
```

Comment:
A typical misuse of a ternary.
Logic like `let variable; if (condition) { variable = vIfConditionTrue } else { variable = vIfConditionFalse }` translates into `let variable = condition ? vIfTrue : vIfFalse` using a ternary